testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
+                            "enumLabels": [
+                                "Continuosly",
+                                "Briefly",
+                                "When the modifier key is pressed"
+                            ],
+                            "default": 2
+                        }
+                    },
+                    // Bitmask setting
+                    "FocusScheme.FocusENHModifier": {
+                        // TODO: Unknown
+                        "schema": {
+                            "title": "Focus Modifiers",
+                            "description": "Bitmask holding 'focus behavior modifiers'.",
+                            "type": "integer"
+                        },
+                        "label": "bitmask"
+                    },
+                    // [EchoTyping]
+                    "EchoTyping.Mode": {
+                        "schema": {
+                            "title": "Keyboard Echo",
+                            "description": "Echoing mode for keyboard.",
+                            "enum": [
+                                2,
+                                32769,
+                                32770,
+                                32771
+                            ],
+                            "enumLabels": [
+                                "No Echo",
+                                "Keys",
+                                "Words",
+                                "Keys and Words"
+                            ],
+                            "default": 32770
+                        }
+                    },
+                    // [EchoMouse]
+                    "EchoMouse.HoverMode": {
+                        "schema": {
+                            "title": "Mouse Echo Hover mode",
+                            "description": "Automatically moves the pointer to menu items and dialog controls when they receive focus.",
+                            "enum": [
+                                0,
+                                1
+                            ],
+                            "enumLabels": [
+                                "Instant mode",
+                                "Hover mode"
+                            ],
+                            "default": 1
+                        }
+                    },
+                    "EchoMouse.Enabled": {
+                        "schema": {
+                            "title": "Mouse Echo",
+                            "description": "Enable/Disables mouse echo.",
+                            "enum": [
+                                0,
+                                1
+                            ],
+                            "enumLabels": [
+                                "Off",
+                                "On"
+                            ],
+                            "default": 1
+                        }
+                    },
+                    "EchoMouse.LineMode": {
```

Comment:
IMO settings keys should be alphabetised.  I can take care of it in the LSR work downstream.

Reply:
Awesome, glad that can happen as part of the automagic LSR Work.